test/box-luatest/gh_8505_synchro_triggers_test.lua

```diff
@@ -0,0 +1,65 @@
+local t = require('luatest')
```

Comment:
This test file belongs to `replication-luatest` suite.

Reply:
Done